SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -802,6 +802,8 @@ int64 USpatialActorChannel::ReplicateActor()
 		}
 	}
 
+	Actor->CallPostReplication();
```

Comment:
Not entirely sure whether this is the place we want to call this.
Additionally, this callback won't be called on native under this implementation.